README.md

```diff
@@ -98,6 +98,11 @@ Known edge cases and other topics are covered in [the FAQ](/FAQ.md).
 Contributions are welcome! Please read the [CONTRIBUTING.md](/CONTRIBUTING.md) document for details
 on getting started.
 
+## Developing
```

Comment:
Remove this section. Reference to prereqs for development are implied in a new prereqs section

Reply:
Carol this is the main ```README``` file which is the main page of the SDK, I included this change because users would know how to navigate to the ```DEVELOPING``` file from the main page.